src/test/groovy/com/palantir/gradle/docker/PalantirDockerPluginTests.groovy

```diff
@@ -141,6 +142,10 @@ class PalantirDockerPluginTests extends AbstractPluginTest {
         execCond("docker rmi -f ${id}")
     }
 
+    // Gradle explicitly disallows the test case, fails with the following:
+    //Could not determine the dependencies of task ':publishDockerPublicationPublicationToMavenLocal'.
+    //> Publishing is not able to resolve a dependency on a project with multiple publications that have different coordinates.
+    @Ignore
```

Comment:
if we think this test is bogus maybe it should just get deleted instead? otherwise I don't know under what criteria is this going to get un-ignored?